aptos-move/move-examples/veiled_coin/sources/sigma_protos.move

```diff
@@ -0,0 +1,884 @@
+/// Package for creating, verifying, serializing & deserializing the $\Sigma$-protocol proofs used in veiled coins.
```

Comment:
i believe the doc generator is unhappy with this comment block, and sigma_protos.md is currently not synced.

Probably because the title symbol `#` is being used multiple times?